README.adoc

```diff
@@ -39,22 +39,64 @@ TIP: Use https://github.com/creationix/nvm/blob/master/README.md[nvm] to manage
 :numbered:
 === Clone the project
 
-First you need to clone the https://github.com/integr8ly/tutorial-web-app-walkthroughs[walkthroughs repository] next to the tutorial web app
+First you need to clone the https://github.com/integr8ly/tutorial-web-app-walkthroughs[walkthroughs repository] next to the tutorial web app:
 
 [source,shell]
 ----
-$ git clone https://github.com/integr8ly/tutorial-web-app-walkthroughs.git
+home:$ git clone https://github.com/integr8ly/tutorial-web-app-walkthroughs.git
+home:$ cd tutorial-web-app-walkthroughs
+home:~/tutorial-web-app-walkthroughs$ install yarn
```

Comment:
Should this be `yarn install`? I wasn't aware we needed any dependencies in the `tutorial-web-app-walkthroughs` repo though. I can double check whether this is required

Reply:
I just did the yarn install as a matter of habit.  I didn't actually test it without doing this. If you have done it previously without then best to take it out.

Reply:
Confirmed that a `yarn install` isn't required here.